fdbclient/NativeAPI.actor.cpp

```diff
@@ -1193,7 +1193,8 @@ Future<Standalone<RangeResultRef>> getRange(
 	KeySelector const& end,
 	GetRangeLimits const& limits,
 	bool const& reverse,
-	TransactionInfo const& info);
+	TransactionInfo const& info,
```

Comment:
Since `getRange` always takes `TransactionInfo` and we `isFetchKeys` is figured from the `TransactionInfo::taskID` do we need to add another parameter here?